app/components/login.jsx

```diff
@@ -1,202 +1,318 @@
+                    onChange={e => methods.onChangePassword(e.target.value)}
+                />
+                <input
+                    name='confirm'
+                    type='password'
+                    placeholder='Confirm Password'
+                    className={cx(classes.input, isLogIn && classes.disabled)}
+                    onChange={e => methods.onChangeConfirm(e.target.value)}
+                />
+                <div className={classes.agreeTermContainer}>
+                    <div className={classes.checkTerm}>
+                        <input
+                            className={classes.checkTermBox}
+                            type='checkbox'
+                            name='agreed'
+                            onClick={e => methods.onChangeAgree(e.target.checked)}
+                        />
+                        <label className={classes.agreeTermLabel}>
+                            I agree to the
+                            <a href='https://enciv.org/terms' target='_blank' className={classes.aLinkTerm}>
+                                Terms of Service
+                            </a>
+                        </label>
+                    </div>
+                </div>
+                <div className={classes.btnContainer}>
+                    <button className={cx(classes.btn, isLogIn && classes.disabled)} onClick={e => methods.signup()}>
+                        Sign Up
+                    </button>
+                    <button className={cx(classes.btn, !isLogIn && classes.disabled)} onClick={e => methods.login()}>
+                        Log In
+                    </button>
+                    <button className={cx(classes.btn, !isLogIn && classes.disabled)} onClick={e => methods.skip()}>
+                        Skip
+                    </button>
+                </div>
+                <div className={cx(classes.resetPasswordBtn, !isLogIn && classes.disabled)}>
+                    <button onClick={e => methods.sendResetPassword()} className={classes.resetBtn}>
+                        Send Reset Password
+                    </button>
+                </div>
+                <div className={cx("go-to-sign-up", isLogIn && classes.disabled)}>
+                    <p> Don't have an account? Click Join, it only takes a minute to sign up!</p>
+                </div>
+                <div>
+                    {state.error && <div style={{ color: '#fec215', textAlign: 'center' }}>{state.error}</div>}
+                    {state.info && <div style={{ color: '#fec215', textAlign: 'center' }}>{state.info}</div>}
+                    {state.success && <div style={{ color: '#fec215', textAlign: 'center' }}>{state.success}</div>}
+                </div>
+            </div>
+        </div>)
 }
 
-export default Login;
+export default React.forwardRef(Login);
+
+const useStyles = createUseStyles(theme => ({
+    Login: {
+        backgroundColor: 'white', 
+        width: '60rem',
```

Comment:
For components we are building, our design pattern is to let the width come down from the parent. That means that this component should fill with width, and not have any margin or padding.  The parent can easily apply it, but it's hard to undo/override if it's there.  So make sure this component renders all the way to the left and right, and top of it's parent in storybook. 

I realize that the component that you started with was for a different design pattern - but this is a pretty easy change.
Don't set 
width, maxWidth, height, left, right, top, bottom, transform, z-index, maxHeight.